Projekty/Projekt1/Koziel_Krupinski_Lis/kamien_milowy_1.ipynb

```diff
@@ -0,0 +1,1572 @@
+{
```

Comment:
<p>Słuszne uwagi. Możliwe że lepszym wyjściem może się okazać usunięcie, &gt;48k wierszy to jest bardzo dużo, więc z tymi 3k można się bez żalu pożegnać. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>